src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/DRConfig/RemoveAzureEventHubDRConfiguration.cs

```diff
@@ -0,0 +1,93 @@
+
+namespace Microsoft.Azure.Commands.EventHub.Commands.GeoDR
+{
+    /// <summary>
+    /// 'Remove-AzureRmEventHubDRConfiguration' Cmdlet Deletes an Alias(Disaster Recovery configuration)
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, EventHubDRConfigurationVerb, SupportsShouldProcess = true, DefaultParameterSetName = AliasPropertiesParameterSet), OutputType(typeof(bool))]
+    public class RemoveEventHubDRConfiguration : AzureEventHubsCmdletBase
+    {
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, Position = 0, HelpMessage = "Resource Group Name")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = AliasPropertiesParameterSet, ValueFromPipelineByPropertyName = true, Position = 1, HelpMessage = "Namespace Name")]
+        [Parameter(Mandatory = true, ParameterSetName = AliasInputObjectParameterSet, ValueFromPipelineByPropertyName = true, Position = 1, HelpMessage = "Namespace Name")]
+        [ValidateNotNullOrEmpty]
+        [Alias(AliasNamespaceName)]
+        public string Namespace { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = AliasPropertiesParameterSet, ValueFromPipelineByPropertyName = true, Position = 2, HelpMessage = "Alias (GeoDR)")]
+        [Parameter(Mandatory = false, ParameterSetName = AliasInputObjectParameterSet, ValueFromPipelineByPropertyName = true, HelpMessage = "Alias (GeoDr)")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = AliasInputObjectParameterSet, ValueFromPipelineByPropertyName = true, Position = 3,HelpMessage = "Alias Configuration object")]        
+        [Parameter(Mandatory = false, ParameterSetName = AliasPropertiesParameterSet, ValueFromPipelineByPropertyName = true, HelpMessage = "Alias Configuration object")]
+        [Alias(AliasAliasObj)]
+        [ValidateNotNullOrEmpty]
+        public PSEventHubDRConfigurationAttributes InputObject { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation")]
+        public SwitchParameter Force { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter PassThru { get; set; }
+
+        public override void ExecuteCmdlet()
+        {            
+            if (ParameterSetName.Equals(AliasInputObjectParameterSet))
+            {                
+                ConfirmAction(
+                Force.IsPresent,
+                string.Format(Resources.DRRemoveAlias, InputObject.Name, Namespace),
+                string.Format(Resources.DRRemoveAlias, InputObject.Name, Namespace),
+                InputObject.Name,
+                () =>
+                {
+                    Client.DeleteEventHubDRConfiguration(ResourceGroupName, Namespace, InputObject.Name);
```

Comment:
I think you would want to get the namespace and resourcegroupname from the inputobject as well as the name.

Reply:
we have removed the inputobject from remove cmdlet.